### PR TITLE
Feature/loggable docker invocations

### PR DESCRIPTION
**Pull request by factyy**
Enable logging of docker supplemental calls: checking image presence and building a new image

### CODE REVIEW

File: lsp-docker.el [hunk 1]
```diff
@@ -405,12 +424,25 @@ Argument DOCKER-CONTAINER-NAME name to use for container."
...
-               (apply #'call-process command-program nil (current-buffer) nil command-arguments)
-               (buffer-string)))))
-      (cons exit-code raw-output))))
+    (progn
+      (lsp-docker--conditionally-log-docker-supplemental-call command-program command-arguments)
+      (lsp-docker--launch-command-internal command-program command-arguments))))
+
+(defun lsp-docker--launch-command-internal (command-program command-arguments)
+  "Run a command using 'call-process' function and return a pair of exit code and raw output"
+  (-let (((exit-code . raw-output)
```
Comment (yyoncho) [inline lsp-docker.el:433]: these locals look redundant. You can return from the temp-buffer call.

**Review by yyoncho — APPROVED** [review]

File: lsp-docker.el (hunk 1 above)
Comment (factyy) [reply lsp-docker.el]: Fixed in subsequent commit

**Comment by factyy** [comment]
@yyoncho could you also take a look at the failing tests? Looks like the failing check is not related to changes in this PR... Nevertheless please approve whether I can merge this PR right now :)

**Comment by factyy** [comment]
Merged as @yyoncho granted approval

**Pull request merged**